### PR TITLE
Feature/styling and final touches

### PR DESCRIPTION
**Pull request by MacCrazyman**
# Final touches.

In this milestone, the project is ready for use.

* Page is fully styled.
* Table order elements automatically.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,24 +4,29 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Bakbak+One&display=swap" rel="stylesheet">
   <title>Leaderboard</title>
 </head>
 <body>
```
Comment (rustam-tolipov) [inline src/index.html:12]: - [ ] Good job so far 👏, but there is something that needs to be done so please kindly fix these issues. In your **README.md** file I recommend changing the usage part where you just said open your index.html file or live version. You can say run `npm install` to install packages, run `npm run build` to create dist folder, and then run `npm start` to start the app 🍀

File: src/manage_scores.js [hunk 2]
```diff
@@ -4,17 +4,22 @@ export default class ScoreLybrary {
...
-      .then((json) => {
-        this.list = json.result;
-      });
+    const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/th5xBYWSs5VtsOmKTkPd/scores/');
+
+    return response.json();
+    // .then((json) => {
+    //   this.list = json.result;
+    //   this.list.sort((prev,next) => next.score - prev.score)
+    // });
```
Comment (rustam-tolipov) [inline src/manage_scores.js]: - [ ] For best practices also consider removing commented code 😊💡

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @MacCrazyman , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (MacCrazyman) [reply src/index.html:12]: Thanks for the kindly suggestions @Rustamxon7

**Comment by helman101** [comment]
# STATUS: APPROVED 🟢 🥳 

Hi @MacCrazyman 🥇 , 

### Congratulations!!! 🥳 🎉

You have successfully completed all the expectations for this project, there is nothing more to do here, now you can go on your way and keep improving your skills.

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**